app/dashboard/DashboardIndex.tsx

```
@@ -36,8 +48,61 @@ const DashboardIndex = () => {
+        <SectionCard title="overdue">
+          {overdue &&
+            overdue.map((task) => (
+              <TaskCard
+                key={task.id}
+                name={task.name}
+                completed={false}
+                tags={["penting", "urgent"]}
+                id="haha"
+                description={task.description}
+              />
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)